languages/rust/exercises/concept/options/tests/options.rs

```diff
@@ -0,0 +1,89 @@
+use options::*;
+
+#[test]
+fn test_reviving_dead_player() {
+    let level: u32 = 34;
+    let dead_player = Player {
+        health: 0,
+        mana: Some(0),
+        level,
+    };
+    let revived_player = dead_player.revive();
+    assert_eq!(revived_player.is_some(), true);
+
+    let revived_player = revived_player.unwrap();
+    assert_eq!(revived_player.health, 100);
+    assert_eq!(revived_player.mana, Some(100));
+    assert_eq!(revived_player.level, level);
+}
+
+#[test]
+#[ignore]
+fn test_reviving_alive_player() {
+    let level: u32 = 8;
+    let alive_player = Player {
+        health: 1,
+        mana: None,
+        level,
+    };
+    assert_eq!(alive_player.revive().is_none(), true);
+}
+
+#[test]
+#[ignore]
+fn test_cast_spell_with_enough_mana() {
+    let health = 99;
+    let mana = 100;
+    let level = 100;
+    let mana_cost = 3;
+
+    let mut accomplished_wizard = Player {
+        health,
+        mana: Some(mana),
+        level,
+    };
+
+    assert_eq!(accomplished_wizard.cast_spell(mana_cost), mana_cost * 2);
+    assert_eq!(accomplished_wizard.health, health);
+    assert_eq!(accomplished_wizard.mana, Some(mana - mana_cost));
+    assert_eq!(accomplished_wizard.level, level);
+}
+
+#[test]
+#[ignore]
+fn test_cast_spell_with_insufficient_mana() {
+    let health = 56;
+    let mana = 0;
```

Comment:
```suggestion
    let mana = 2;
```

By using a non-0 value, we reduce the chance that a student's accidentally done the right thing by applying a default somewhere.